spec/lib/declare_schema/generator_spec.rb

```diff
@@ -88,7 +98,11 @@ class Alpha::BetaTest < ActiveSupport::TestCase
     expect(File.exist?('db/schema.rb')).to be_truthy
 
     if defined?(SQLite3)
-      expect(File.exist?("db/development.sqlite3") || File.exist?("db/test.sqlite3")).to be_truthy
+      if ActiveSupport.version >= Gem::Version.new('7.1.0')
+        expect(File.exist?("storage/development.sqlite3") || File.exist?("storage/test.sqlite3")).to be_truthy
```

Comment:
📓 Rails 7.1 moved the default location of sqlite databases into the `storage/` directory to make gitignoring easier.